src/js/view/components/timeslider.js

```diff
@@ -207,6 +211,18 @@ define([
             }
 
             var timetipText;
+
+            // With touch events, we never will get the hover events on the cues that cause cues to be active.
+            // Therefore use the info we about the scroll position to detect if there is a nearby cue to be active.
+            if (evt.sourceEventType === 'touch') {
```

Comment:
I don't believe we can do something like that.  The problem is that a touch event always targets the original element that it first targeted.  There isn't really a listener that can be put on those that will recieve the touchmove events that result from the drag.